frontend/components/interests/form/InterestFormTextInput.tsx

```diff
@@ -24,6 +25,7 @@ const InterestFormTextInput = ({
   multiline,
   InputProps,
   minRows,
+  disableAutocomplete,
 }: TextInputProps) => {
   const [, { value, error, touched }, { setTouched, setValue }] = useField(name);
```

Comment:
Why is there an extra `,`?

Reply:
What do you mean?  The last item in an opened braces will always have a trailing comma according to Prettier, right?
```
{
  ....
  x,
}
```

Reply:
I see